### PR TITLE
Shuttle services information

### LINKED ISSUE #18: US2F : As a user, I want information about the Concordia Shuttle services.
The information includes the shuttle locations, schedule as well as directions to the shuttle from the current location

### PR DESCRIPTION
**Pull request by trevornag**
- Show the user schedule information for the shuttle bug (SGW and LOY)
- Give the user the path on the map from their current location to the shuttle bus stop (they will select SGW or LOY)

### CODE REVIEW

**Review by gisellemartel — CHANGES_REQUESTED** [review]
Please remove npm package manager dependency. There is currently both a package-lock.json (see attached screenshot) and yarn.lock file, which could cause dependency conflicts. 
Screen Shot 2020-02-29 at 6 48 05 PM

**Comment by trevornag** [comment]
> Please remove npm package manager dependency. There is currently both a package-lock.json (see attached screenshot) and yarn.lock file, which could cause dependency conflicts.
> Screen Shot 2020-02-29 at 6 48 05 PM

Sorry about that. Yes, I agree this should be deleted. It was added because while adding react libraries. Will remove it

File: components/map/index.js [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
-import MapView, { Polygon, PROVIDER_GOOGLE } from 'react-native-maps';
+import { View } from 'react-native';
+import MapView, { Polygon, Polyline, PROVIDER_GOOGLE } from 'react-native-maps';
 import buildings from '../../assets/polygons/polygons';
 import styles from './styles';
 
 export default class TheMap extends Component {
   constructor(props) {
     super(props);
+    const waypoints = '';
```
Comment (gisellemartel) [inline components/map/index.js]: Is this really needed? Could you not just initialize like this instead: `coordinates: ''`

File: components/map/index.js [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
     this.state = {
       region: {
         latitude: 45.492409,
         longitude: -73.582153
       },
+      coordinates: waypoints,
```
Comment (gisellemartel) [inline components/map/index.js]: see previous comment

File: components/searchBar/index.js [hunk 3]
```diff
@@ -57,7 +58,7 @@ export default class searchBar extends Component {
       const georesult = await fetch(geoUrl);
       const gjson = await georesult.json();
       const locations = gjson.result.geometry.location;
-      console.log(gjson.result.geometry.location);
+      // console.log(gjson.result.geometry.location);
```
Comment (gisellemartel) [inline components/searchBar/index.js]: is this needed?

File: components/searchBar/index.js [hunk 4]
```diff
@@ -66,7 +67,7 @@ export default class searchBar extends Component {
           longitudeDelta: 0.05
         }
       });
-      console.log(this.state.region);
+      // console.log(this.state.region);
```
Comment (gisellemartel) [inline components/searchBar/index.js]: is this needed?

File: components/shuttleInformation/index.js [hunk 5]
```diff
@@ -0,0 +1,139 @@
+/* eslint-disable no-console */
```
Comment (gisellemartel) [inline components/shuttleInformation/index.js:1]: can some of these be disabled globally in eslintr.json?

File: components/shuttleInformation/shuttleScheduleService.js [hunk 6]
```diff
@@ -0,0 +1,140 @@
+/* eslint-disable no-unused-vars */
```
Comment (gisellemartel) [inline components/shuttleInformation/shuttleScheduleService.js:1]: Should perhaps leave a comment that this is being disabled temporarily until the module that uses this variable will be created later. Should be deleted later

File: components/map/index.js (hunk 1 above)
Comment (trevornag) [reply components/map/index.js]: Nice catch! I will fix that

File: components/shuttleInformation/index.js (hunk 5 above)
Comment (trevornag) [reply components/shuttleInformation/index.js:1]: Yes, this is something to consider perhaps in the future as it is re-occurring

File: components/map/index.js (hunk 1 above)
Comment (trevornag) [reply components/map/index.js]: in commit [#18] Fix minor coding issues (useless variables and console.log)

**Review by gisellemartel — APPROVED** [review]
Thank you for making the requested changes. Code looks good and functions as intended. Approved.

**Review by GD-Dheer — APPROVED** [review]
Code is clean and tested both parts. Good to go

**Review by TalalBazerbachi — APPROVED** [review]
The functionality works and the code looks clean.
I think it's ready to be merged.

**Pull request merged**